### ISSUE TITLE
Accuracy and loss calculation

### ISSUE THREAD

**Issue by mistaleee** [TuSimple/simpledet#205]
Hi all, 

I'm currently deep diving in the code of Tridentnet to find out why the RpnL1 and RcnnL1 losses become bigger while the accuracy rates become smaller. I tried out the learning rates 0.0001, 0.0005, 0.00001, 0.00005, 0.000001, 0.000005, but the behavior is the same for all learning rates (see #196 ).

Therefore, it would be helpful to know how the losses are calculated (I already found the calculation of the accuracies). 

Thanks for any hints!

**Comment by RogerChern** [comment]
You can find it here. Basically, we give the moving average of l1 loss for all foreground objects both in RPN and RCNN. Unlike Detectron, we do not divide the total l1 loss by a fixed number(256 or else), instead, we report the real mean of L1.

https://github.com/TuSimple/simpledet/blob/962cf2e9f1335f18ea718ded90c8020d20352166/core/detection_metric.py#L117-L139

**Comment by mistaleee** [comment]
Thanks  @RogerChern .
So, do you just some over the difference between correctly and incorrectly predicted samples? Or, is there a formula?

**Comment by RogerChern** [comment]
Sorry for the late reply. It seems that the lr scheduler is working properly during finetune. We will fix the issue and give a detailed tutorial about finetuning.

**Comment by mistaleee** [comment]
@RogerChern thanks for your great and timely support.
